lib/PuppeteerSharp/Page.cs

```diff
@@ -310,6 +310,32 @@ public class Page : IDisposable
 
         #region Public Methods
 
+        /// <summary>
+        /// Sets the page's geolocation.
+        /// </summary>
+        /// <returns>The task.</returns>
+        /// <param name="options">Geolocation options.</param>
+        /// <remarks>
+        /// Consider using <seealso cref="BrowserContext.OverridePermissionsAsync(string, IEnumerable{OverridePermission})"/> to grant permissions for the page to read its geolocation.
+        /// </remarks>
+        public async Task SetGeolocationAsync(GeolocationOption options)
```

Comment:
this method doesn't have to use the `async` keyword, just return the last statement